src/plugins/usrbg/index.tsx

```diff
@@ -104,24 +108,42 @@ export default definePlugin({
 
     useBannerHook({ displayProfile, user }: any) {
         if (displayProfile?.banner && settings.store.nitroFirst) return;
-        if (data[user.id]) return data[user.id];
+        if (this.userHasBackground(user.id)) return this.getImageUrl(user.id);
     },
 
     premiumHook({ userId }: any) {
-        if (data[userId]) return 2;
+        if (this.userHasBackground(userId)) return 2;
     },
 
     shouldShowBadge({ displayProfile, user }: any) {
-        return displayProfile?.banner && (!data[user.id] || settings.store.nitroFirst);
+        return displayProfile?.banner && (!this.userHasBackground(user.id) || settings.store.nitroFirst);
+    },
+
+    userHasBackground(userId: string) {
+        if (this.data === null) return false;
+        return this.data.users[userId] !== undefined;
```

Comment:
could just change this entire func to
```suggestion
        return !!this.data?.users[userId];
```